src/ker_horner_allw_loop_constexpr.c

```diff
@@ -0,0 +1,205 @@
+// Code generated by gen_all_horner_C_code.m in finufft/devel
+// Authors: Alex Barnett & Ludvig af Klinteberg.
+// (C) The Simons Foundation, Inc.
+  if constexpr(w==2) {
```

Comment:
same as the other version the GPU will use this, no need for padding. CPU will use the new format in any case.